### ISSUE TITLE
Audio devices pop-over menu is to small

### ISSUE THREAD

**Issue by noraj** [jitsi/jitsi-meet#12595]
### Description:

The pop-over menu containing the list of audio devices (microphones and speakers) has a fixed width that is small. The consequence is that many devices with a long name are cropped and it becomes impossible to tell which device is what.

### Steps to reproduce:

1. Launch a Jitsi Meet conference
2. Open the audio device menu (little arrow in the top right corner of the microphone button).
3. Observe the issue if you have devices with a long name.

### Expected behavior:

Having the device popup automatically resizing depending on the devices name or at least having a way longer max-width (eg.900px, not a fixed width).

![image](https://user-images.githubusercontent.com/16578570/203087979-d5f7d7ef-24f1-4740-b005-5827f90cc98a.png)

### Actual behavior:

The mic and speak list has a fixed (300px it seems) width and crop the name of the devices which make it really difficult to identify the correct devices.

![image](https://user-images.githubusercontent.com/16578570/203085083-88078ac3-4d2c-48bc-befa-bfaa1d83abda.png)

### Server information:

N/A, can also test with https://meet.jit.si/

### Client information:

- Browser / app version:
  - Chromium Version 107.0.5304.110 (Official Build) Arch Linux (64-bit)
- Operating System: ArchLinux

**Comment by VernPace** [comment]
I agree. This is the biggest complaint of our users. In addition, there is redundancy of devices listed - the same device is listed three times as shown below where both the microphone and speakers are prefaced with "Communication -" and "Same as system". Is this really neccessary?

![Capture](https://user-images.githubusercontent.com/94239732/203140641-d2eb487c-5779-4edf-8cac-b508131ebb96.PNG)

**Comment by saghul** [comment]
Those labels come straight from the browser.

**Comment by noraj** [comment]
> I agree. This is the biggest complaint of our users. In addition, there is redundancy of devices listed - the same device is listed three times as shown below where both the microphone and speakers are prefaced with "Communication -" and "Same as system". Is this really neccessary?

Seems normal, on laptop you can have a headset that plays both the role of a speaker / headphone and microphone. But anyway this is offtopic for this issue.

**Comment by JAYESHBATRA** [comment]
Please review the commit and approve if the issue is fixed now.

**Comment by saghul** [comment]
@quitrk didn't you fix this do it would scroll?

**Comment by quitrk** [comment]
@saghul That's correct. It was fixed [here](https://github.com/jitsi/jitsi-meet/commit/baf5aa14e), however it doesn't seem to be deployed yet in prod, I do see the fix on alpha however

**Comment by saghul** [comment]
Fixed in https://github.com/jitsi/jitsi-meet/pull/12974
